tests/test.rs

```diff
@@ -2158,6 +2158,12 @@ fn test_high_byte_strings_and_unicode_strings_without_reserved_tags() {
             }
         }
     }
+    // FIXME: Libreoffice recognizes a REPT("O", I44) formula
```

Comment:
Do you have a sample file with the unrecognised tag?

Reply:
The assert below attempts to retrieve the value, it does not recognize the formula, is this not what you meant?

Reply:
I was asking if you have a test file with this tag?

Reply:
192 is 0x00C0 = 'TOOLBAREND'. I'm just not clear if that is the context it is appearing in here.

It would normally be accompanied by 0x00BF => 'TOOLBARHDR'.

Reply:
This set of commits introduced this test and the test file.